<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SenderVideoDialogClass</class>
 <widget class="QDialog" name="SenderVideoDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>979</width>
    <height>780</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="VideoWidget" name="videoWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>320</width>
       <height>240</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="shutterLabel">
       <property name="text">
        <string>Shutter:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QSlider" name="shutterSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="6">
      <widget class="QSlider" name="vrSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSlider" name="gainSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QLabel" name="vrLabel">
       <property name="text">
        <string>VR:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="gainLabel">
       <property name="text">
        <string>Gain:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLabel" name="uvLabel">
       <property name="text">
        <string>UV:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QSlider" name="uvSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="wbLabel">
       <property name="text">
        <string>White balance =&gt; </string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="fpsLabel">
       <property name="text">
        <string>FPS</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>VideoWidget</class>
   <extends>QWidget</extends>
   <header>videowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>shutterSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>SenderVideoDialogClass</receiver>
   <slot>onShutterChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>389</x>
     <y>759</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>389</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gainSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>SenderVideoDialogClass</receiver>
   <slot>onGainChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>863</x>
     <y>759</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>389</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>uvSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>SenderVideoDialogClass</receiver>
   <slot>onUVChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>875</x>
     <y>720</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>389</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vrSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>SenderVideoDialogClass</receiver>
   <slot>onVRChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>875</x>
     <y>755</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>389</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onShutterChanged(int)</slot>
  <slot>onGainChanged(int)</slot>
  <slot>onUVChanged(int)</slot>
  <slot>onVRChanged(int)</slot>
 </slots>
</ui>
